var/spack/repos/builtin/packages/dealii/package.py

```diff
@@ -87,79 +88,83 @@ class Dealii(CMakePackage, CudaPackage):
-    depends_on("assimp",           when='@9.0:+assimp')
-    depends_on("doxygen+graphviz", when='+doc')
-    depends_on("graphviz",         when='+doc')
-    depends_on("gsl",              when='@8.5.0:+gsl')
-    depends_on("hdf5+mpi+hl",      when='+hdf5+mpi')
-    depends_on("cuda@8:",          when='+cuda')
-    depends_on("cmake@3.9:",       when='+cuda')
+    depends_on('mpi',              when='+mpi')
+    depends_on('adol-c@2.6.4:',    when='@9.0:+adol-c')
+    depends_on('arpack-ng+mpi',    when='+arpack+mpi')
+    depends_on('assimp',           when='@9.0:+assimp')
+    depends_on('doxygen+graphviz', when='+doc')
+    depends_on('graphviz',         when='+doc')
+    depends_on('gsl',              when='@8.5.0:+gsl')
+    depends_on('hdf5+mpi+hl',      when='+hdf5+mpi')
+    depends_on('cuda@8:',          when='+cuda')
+    depends_on('cmake@3.9:',       when='+cuda')
     # currently deal.II does not build with Cmake 3.10, see
     # https://github.com/dealii/dealii/issues/5510
-    depends_on("cmake@:3.9.99")
+    depends_on('cmake@:3.9.99')
     # FIXME: concretizer bug. The two lines mimic what comes from PETSc
     # but we should not need it
-    depends_on("metis@5:+int64+real64",   when='+metis+int64')
-    depends_on("metis@5:~int64+real64",   when='+metis~int64')
-    depends_on("nanoflann",        when="@9.0:+nanoflann")
-    depends_on("netcdf+mpi",       when="+netcdf+mpi")
-    depends_on("netcdf-cxx",       when='+netcdf+mpi')
-    depends_on("oce",              when='+oce')
-    depends_on("p4est",            when='+p4est+mpi')
-    depends_on("petsc+mpi~int64",  when='+petsc+mpi~int64')
-    depends_on("petsc+mpi+int64",  when='+petsc+mpi+int64')
-    depends_on("petsc@:3.6.4",     when='@:8.4.1+petsc+mpi')
+    depends_on('metis@5:+int64+real64',   when='+metis+int64')
+    depends_on('metis@5:~int64+real64',   when='+metis~int64')
+    depends_on('nanoflann',        when='@9.0:+nanoflann')
+    depends_on('netcdf+mpi',       when='+netcdf+mpi')
+    depends_on('netcdf-cxx',       when='+netcdf+mpi')
+    depends_on('oce',              when='+oce')
+    depends_on('p4est',            when='+p4est+mpi')
+    depends_on('petsc+mpi~int64',  when='+petsc+mpi~int64')
+    depends_on('petsc+mpi+int64',  when='+petsc+mpi+int64')
+    depends_on('petsc@:3.6.4',     when='@:8.4.1+petsc+mpi')
     depends_on('python',           when='@8.5.0:+python')
-    depends_on("slepc",            when='+slepc+petsc+mpi')
-    depends_on("slepc@:3.6.3",     when='@:8.4.1+slepc+petsc+mpi')
-    depends_on("slepc~arpack",     when='+slepc+petsc+mpi+int64')
-    depends_on("sundials~pthread", when='@9.0:+sundials')
-    depends_on("trilinos+amesos+aztec+epetra+ifpack+ml+muelu+sacado+teuchos",       when='+trilinos+mpi~int64')
-    depends_on("trilinos+amesos+aztec+epetra+ifpack+ml+muelu+sacado+teuchos~hypre", when="+trilinos+mpi+int64")
+    depends_on('scalapack',        when='@9.0:+nanoflann')
```

Comment:
Oh, I suppose that this should be 
`depends_on('scalapack',        when='@9.0:+scalapack')`

Reply:
Absolutely, will fix